src/platform/Darwin/BleConnectionDelegateImpl.mm

```diff
@@ -33,12 +33,15 @@
 
 using namespace chip::Ble;
 
+#define SCANNING_TIMEOUT_IN_SECONDS 60ull
```

Comment:
does the objective C support `constexpr uint64_t kScanningTimeoutInSeconds = 60` for typesafety without the ull suffix?
Does this need to be ull ?